tests/test_api_urls.py

```diff
@@ -71,6 +71,19 @@ def test_api_labels_without_auth(self):
         self.assertEqual(rv.status_code, 200)
         self.assertEqual(rv.content_type, 'application/json')
 
+    def test_api_comments_link_header_auth(self):
+        '''API access to comments greater than 30 returns pagination in Link header of the response.'''
+        query_string = {'callback': 'foo'}
```

Comment:
The test tries to find if the link header property is there in the response header for 
1) Number of comments > 30 (AssertTrue)
2) Number  of comments < 30 (Assertion Not True)

When I use self.app.get(/api/issues/398/comments', environ_base=headers)

In my response I do not get any header meta information. The response looks like this: 

`    [
  {
    "url": "https://api.github.com/repos/webcompat/webcompat-tests/issues/comments/223817202",
    "html_url": "https://github.com/webcompat/webcompat-tests/issues/398#issuecomment-223817202",
    "issue_url": "https://api.github.com/repos/webcompat/webcompat-tests/issues/398",
    "id": 223817202,
    "user": {
      "login": "deepthivenkat",
      "id": 7373087,
      "avatar_url": "https://avatars.githubusercontent.com/u/7373087?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/deepthivenkat",
      "html_url": "https://github.com/deepthivenkat",
      "followers_url": "https://api.github.com/users/deepthivenkat/followers",
      "following_url": "https://api.github.com/users/deepthivenkat/following{/other_user}",
      "gists_url": "https://api.github.com/users/deepthivenkat/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/deepthivenkat/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/deepthivenkat/subscriptions",
      "organizations_url": "https://api.github.com/users/deepthivenkat/orgs",
      "repos_url": "https://api.github.com/users/deepthivenkat/repos",
      "events_url": "https://api.github.com/users/deepthivenkat/events{/privacy}",
      "received_events_url": "https://api.github.com/users/deepthivenkat/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-06-05T14:49:09Z",
    "updated_at": "2016-06-05T14:49:09Z",
    "body": "1"
  }]`

So I try to make a [JSONP callback](http://flask-restless.readthedocs.io/en/stable/requestformat.html#jsonp-callbacks) to have the JSON format of the response- here I get the meta information:

`
/**/foo({
  "meta": {
    "Content-Type": "application/javascript; charset=utf-8",
    "X-RateLimit-Limit": "5000",
    "X-RateLimit-Remaining": "4995",
    "X-RateLimit-Reset": "1470066069",
    "Cache-Control": "private, max-age=60, s-maxage=60",
    "Vary": "Accept, Authorization, Cookie, X-GitHub-OTP",
    "ETag": "\"e1da1eadb2b4a7e4a726ca503e451706\"",
    "X-OAuth-Scopes": "admin:org, admin:org_hook, admin:public_key, admin:repo_hook, delete_repo, gist, notifications, repo, user",
    "X-Accepted-OAuth-Scopes": "",
    "X-GitHub-Media-Type": "github.v3",
    "Link": [
      [
        "https://api.github.com/repositories/17839063/issues/398/comments?callback=foo&page=2",
        {
          "rel": "next"
        }
      ],
      [
        "https://api.github.com/repositories/17839063/issues/398/comments?callback=foo&page=4",
        {
          "rel": "last"
        }
      ]
    ],
    "status": 200
  },
  "data": [
    {
      "url": "https://api.github.com/repos/webcompat/webcompat-tests/issues/comments/223817202",
      "html_url": "https://github.com/webcompat/webcompat-tests/issues/398#issuecomment-223817202",
      "issue_url": "https://api.github.com/repos/webcompat/webcompat-tests/issues/398",
      "id": 223817202,
      "user": {
        "login": "deepthivenkat",
        "id": 7373087,
        "avatar_url": "https://avatars.githubusercontent.com/u/7373087?v=3",
        "gravatar_id": "",
        "url": "https://api.github.com/users/deepthivenkat",
        "html_url": "https://github.com/deepthivenkat",
        "followers_url": "https://api.github.com/users/deepthivenkat/followers",
        "following_url": "https://api.github.com/users/deepthivenkat/following{/other_user}",
        "gists_url": "https://api.github.com/users/deepthivenkat/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/deepthivenkat/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/deepthivenkat/subscriptions",
        "organizations_url": "https://api.github.com/users/deepthivenkat/orgs",
        "repos_url": "https://api.github.com/users/deepthivenkat/repos",
        "events_url": "https://api.github.com/users/deepthivenkat/events{/privacy}",
        "received_events_url": "https://api.github.com/users/deepthivenkat/received_events",
        "type": "User",
        "site_admin": false
      },
      "created_at": "2016-06-05T14:49:09Z",
      "updated_at": "2016-06-05T14:49:09Z",
      "body": "1"
    }]`

where I can determine if linkheader property is being sent in response by github or not.